### DISCUSSION TITLE
Reuters RSS Feed incompatible?

### DISCUSSION THREAD

**Discussion by ANDREW007CZ** [spacecowboy/Feeder#335] [Q&A] [answered]
Hi,
anyhow I try, I can't import a rss channel from Reuters.
https://www.reutersagency.com/en/reutersbest/reuters-best-rss-feeds/

Does anyone know how to do that, please? There is no simple URL, but a some sort of XML is downloaded instead. That can't be imported to Feeder.
Thanks!

**Comment by BugFixNo1** [comment]
also not working: http://www.docma.info/rss.xml

@spacecowboy -> it would be smart to have a general posting with incompatible feeds to solve such things and getting the app mor universal

**Reply by markcellus** [reply]
http://www.docma.info/rss.xml is an invalid RSS feed, so the docma.info website owner will need to fix that.

**Reply by BugFixNo1** [reply]
U R right, thx. New feed adress is https://www.docma.info/feed

**Comment by markcellus** [comment, marked as answer]
Unfortunately Reuters feeds are unreliable and most won't work. You can try Reuters feeds from openrss.org that provide RSS feeds for each section on the reuters.com website. Here's an RSS feed for the World/US section, for example.

https://openrss.org/www.reuters.com/world/us/

**Comment by ANDREW007CZ** [comment]
Thank you, guys! :)
